TcUnit-Runner/AutomationInterface.cs

```diff
@@ -14,7 +14,7 @@ namespace TcUnit.TcUnit_Runner
     /// </summary>
     class AutomationInterface
     {
-        private ITcSysManager10 sysManager = null;
+        private ITcSysManager13 sysManager = null;
```

Comment:
Is ITcSysManager13 also available in TwinCAT 4020? (TcUnit supports > 4020.0).
Is there anything specific that needs 13 that 10 doesn't have?

Reply:
I am not sure if ITcSysManager13 is available in TwinCAT 4020. 
I do not have version 4020.0 for validation

In 13 you can read out via the SysManger interface whether the version is pinned.

In the current version ITcSysManager13 and sysManager.TcVersionFixed is not used.
can therefore be removed and remain on ITcSysManager10.
`        public bool TcPinnedVersion
        {
            get
            {
                return sysManager.TcVersionFixed;
            }
        }`

Why is sysManager.TcVersionFixed not used? I would like to post something about the general procedure below

Reply:
It has to work with 4020.0, and it has to be validated to work with 4020.0 (as it does now). You can get 4020.0 from the Beckhoff support, or download it via this link: https://www.beckhoff.com/forms/twincat3/warenkorb.aspx?lg=en&title=TC31-Full-Setup.3.1.4020.0&version=3.1.4020.0